packages/typo3-docs-theme/assets/sass/_text.scss

```diff
@@ -0,0 +1,7 @@
+/* Bootstrap 5 tweak: do not underline links unless hovered over */
+a:not([class*="btn"]) {
+    text-decoration: none;
+}
+a:not([class*="btn"]):hover {
```

Comment:
```suggestion
a:not([class*="btn"]):hover,
a:not([class*="btn"]):focus {
```

This way it the underline is also displayed when using a keyboard.

Reply:
I supposed that underlined text in that black focus box do not really improve readability?

![image](https://github.com/TYPO3-Documentation/render-guides/assets/48202465/ed1be747-f6e9-4782-b45e-658e42558526)

Reply:
It is also not used currently:
![image](https://github.com/TYPO3-Documentation/render-guides/assets/48202465/73e9d150-89b1-4981-aad1-16cb45c86afc)

Reply:
Ah, so we have an outline. Okay then for me, sorry, haven't checked.